src/migrations/1689777747530-default-item-settings.ts

```diff
@@ -1,14 +1,13 @@
-import { MigrationInterface, QueryRunner } from "typeorm";
+import { MigrationInterface, QueryRunner } from 'typeorm';
 
 export class Migrations1689777747530 implements MigrationInterface {
-    name = 'Migrations1689777747530';
+  name = 'Migrations1689777747530';
 
-    public async up(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query(`ALTER TABLE "item" ALTER COLUMN "settings" DROP DEFAULT`);
-    }
-
-    public async down(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query(`ALTER TABLE "item" ALTER COLUMN "settings" SET DEFAULT '{}'`);
-    }
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "item" ALTER COLUMN "settings" DROP DEFAULT`);
```

Comment:
This should not be in your PR I think, it should already be done on the last migration on main, make sure to run existing migration before running generate

Reply:
@morganridel this is Prettier's work, because the files were not formatted properly in #494. By the way note that I've added `yarn check` in this PR to avoid this same problem in the future (the CI will fail if prettier was not applied):

https://github.com/graasp/graasp/blob/b004979795d4aa598e13f8704ccf898099fbdd27/.github/workflows/test.yml#L213-L215

Reply:
alright